packages/harmony/tsconfig.json

```diff
@@ -4,22 +4,24 @@
     "lib": ["ES2020", "dom"],
     "module": "esnext",
     "checkJs": false,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
```

Comment:
and i guess same question why this is not needed in stems?

Reply:
stems doesn't have the autocomplete, is very lame